src/excmds.ts

```diff
@@ -3095,18 +3095,18 @@ export function yank(...content: string[]) {
 //#background_helper
 async function setclip(data: string) {
     // Function to avoid retyping everything everywhere
-    const setclip_selection = () => Native.clipboard("set", data)
+    const setclip_selection = data => Native.clipboard("set", data)
 
-    let promises: Array<Promise<any>> = []
+    let promises: Promise<any>[]
```

Comment:
The reason I used `Array<>` instead of `[]` was because we have a linter rule that prohibits `[]` on complex types or something like that. Maybe we should get rid of that?

Reply:
Oh, yeah, I'll just turn it off.